sdk/src/sharedTest/java/io/customer/sdk/repository/ProfileRepositoryTest.kt

```diff
@@ -67,6 +67,29 @@ class ProfileRepositoryTest : BaseTest() {
         verifyNoMoreInteractions(backgroundQueueMock)
     }
 
+    @Test
+    fun identify_givenEmptyIdentifier_expectNoIdentifyBackgroundQueue() {
```

Comment:
Should we also add test to expect no updates in profile for empty identifier when it is already identified?